articles/cognitive-services/text-analytics/quickstarts/python.md

```diff
@@ -1,420 +1,239 @@
+        text="I had a wonderful experience! The rooms were wonderful and the staff was helpful."),
+    models.MultiLanguageInput(id="2", language="en", 
+        text="I had a terrible time at the hotel. The staff was rude and the food was awful."),
+    models.MultiLanguageInput(id="3", language="es", 
+        text="Los caminos que llevan hasta Monte Rainier son espectaculares y hermosos."),
+    models.MultiLanguageInput(id="4", language="es", 
+        text="La carretera estaba atascada. Había mucho tráfico el día de ayer."),
+]
+
+key_phrases_results = client.key_phrases(documents=sentiment_docs)
 ```
 
-The JSON object can be rendered as an HTML table using the following lines of code:
+Much like we've seen with other methods, after the `key_phrases` call, `key_phrases_results.documents` is a list of `KeyPhraseBatchResultItem` instances, each corresponding to a submitted document. The `KeyPhraseBatchResultItem` has a `key_phrases` attribute, which is the detected sentiment value. The Python code below displays the results as an HTML table.
 
 ```python
+from IPython.display import HTML  # or HTML = print if you aren't using a Jupyter notebook
+
 table = []
-for document in key_phrases["documents"]:
-    text    = next(filter(lambda d: d["id"] == document["id"], documents["documents"]))["text"]    
-    phrases = ",".join(document["keyPhrases"])
-    table.append("<tr><td>{0}</td><td>{1}</td>".format(text, phrases))
-HTML("<table><tr><th>Text</th><th>Key phrases</th></tr>{0}</table>".format("\n".join(table)))
-```
+header = "<tr><th>{}</th><th>{}</th><th>{}</th></tr>".format("ID", "Text", "Key phrases")
 
-The next few lines of code call out to the language detection API using the `requests` library in Python to determine the language in the documents.
-```python
-headers   = {'Ocp-Apim-Subscription-Key': subscription_key}
-response  = requests.post(key_phrase_api_url, headers=headers, json=documents)
-key_phrases = response.json()
-pprint(key_phrases)
-```
+for doc, res in zip(key_phrases_docs, key_phrases_results.documents):
+    phrases = ",".join(res.key_phrases)
+    row = "<tr><td>{doc.id}</td><td>{doc.text}</td><td>{phrases}</td></tr>".format(doc=doc, phrases=phrases)
+    table.append(row)
 
-Successful JSON response:
-```json
-{'documents': [
-    {'keyPhrases': ['wonderful experience', 'staff', 'rooms'], 'id': '1'},
-    {'keyPhrases': ['food', 'terrible time', 'hotel', 'staff'], 'id': '2'},
-    {'keyPhrases': ['Monte Rainier', 'caminos'], 'id': '3'},
-    {'keyPhrases': ['carretera', 'tráfico', 'día'], 'id': '4'}],
-    'errors': []
-}
+HTML("<table>{0}{1}</table>".format(header, "\n".join(table)))
 ```
 
-## Identify entities
+## Identify linked entities
+
+Finally, the [`entities` method](https://westus.dev.cognitive.microsoft.com/docs/services/TextAnalytics.V2.0/operations/5ac4251d5b4ccd1554da7634) identifies entities (businesses, people, places, and other proper nouns) in a text document.
```

Comment:
We need this to point to the latest API version.
https://westus.dev.cognitive.microsoft.com/docs/services/TextAnalytics-v2-1/operations/56f30ceeeda5650db055a3c7

Reply:
IMHO these links actually need to point to the SDK documentation, not the REST API documentation. I have done this. I did update the link earlier in the document to the REST documentation and called it out as such.